.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "vyper/vyper-go"]
+	path = vyper/vyper-go
+	url = https://github.com/mac-lawson/vyper-go.git
```

Comment:
This looks like a private repo, needs to be public. 

Also, can we just use this library as a dependency instead of doing this submodule?

Reply:
I'm just going to remove it all together, as it ended up not being needed.